docs/csharp/features.md

```diff
@@ -1,27 +0,0 @@
----
```

Comment:
Now that the content is live, do we want to set up redirection for topics we're deleting or renaming? There's a way to do it, but it requires you to keep the old file there (you just need to add some metadata to it) :cry: 
I haven't done myself but I've seen the instructions on internal guidance sites